components/CorrectCounter/CorrectCounter.js

```diff
@@ -0,0 +1,78 @@
+import styled from "styled-components";
+import RoundButton from "../Buttons/RoundButton";
+import MarkAsIncorrect from "@/public/icons/MarkAsIncorrect.svg";
+import MarkAsCorrect from "@/public/icons/MarkAsCorrect.svg";
+import Stack from "@/public/icons/Stack.svg";
+import { useRouter } from "next/router";
+
+export default function CorrectCounter({
+  variant,
+  actionMode,
+  getIncorrectFlashcardsFromCollection,
+  getCorrectFlashcardsFromCollection,
+  id,
+  isFooter,
+}) {
+  const router = useRouter();
+
+  if (variant !== "correct" && variant !== "incorrect" && variant !== "all") {
+    return console.log("This variant doesn't exist.");
+  }
```

Comment:
What is the purpose of this if-statement?

Reply:
I thought this was just error handling. It could help developers to identify the error if an incorrect or no variant is provided. Would you leave that out?

Reply:
Yes, I would leave this out, since it does not really provide a benefit to your users.
I am assuming this will only apply in edge cases anyways: The if-statement has to be true and a user would need to open the dev tools to see the message, which most likely would not be the case for the average user.

Reply:
Yes, it's not an advantage for the user, I was thinking more of an advantage for a future developer. But I'll delete it. Thanks for your quick reply! 🙏